requirements/edx/github.txt

```diff
@@ -39,10 +39,10 @@ git+https://github.com/hmarr/django-debug-toolbar-mongo.git@b0686a76f1ce3532088c
 -e git+https://github.com/edx/event-tracking.git@0.2.0#egg=event-tracking
 -e git+https://github.com/edx-solutions/django-splash.git@7579d052afcf474ece1239153cffe1c89935bc4f#egg=django-splash
 -e git+https://github.com/edx/acid-block.git@e46f9cda8a03e121a00c7e347084d142d22ebfb7#egg=acid-xblock
--e git+https://github.com/edx/edx-ora2.git@release-2015-07-09T15.00#egg=edx-ora2
+-e git+https://github.com/edx/edx-ora2.git@db/update-lxml#egg=edx-ora2
 -e git+https://github.com/edx/edx-submissions.git@7c766502058e04bc9094e6cbe286e949794b80b3#egg=edx-submissions
 -e git+https://github.com/edx/opaque-keys.git@27dc382ea587483b1e3889a3d19cbd90b9023a06#egg=opaque-keys
-git+https://github.com/edx/ease.git@release-2015-07-09#egg=ease==0.1.3
+git+https://github.com/edx/ease.git@db/update-lxml#egg=ease==0.1.3
```

Comment:
I take it this is your personal branch? Can we do a more formal release, or is the idea to do this quickly to unblock the platform release and then do a release shortly afterwards. I don't like having personal branches referenced in here. I think it would be clearer with a commit hash instead, because then at least it won't break if you clean up your private branches at some point.